### PR TITLE
Add control overlay and notification section

### PR DESCRIPTION
**Pull request by Ryooshuu**

### CODE REVIEW

File: Kumi.Game/KumiGame.cs [hunk 1]
```diff
@@ -89,6 +93,16 @@ protected override void LoadComplete()
...
         Task.Factory.StartNew(() =>
         {
+            // This has to be manually added first because of some weird fuckery i can't explain at the moment
+            LoadComponent(ControlOverlay = new ControlOverlay
+            {
+                Anchor = Anchor.TopRight,
+                Origin = Anchor.TopRight
+            });
+            
+            DependencyContainer.CacheAs<INotificationManager>(ControlOverlay);
```
Comment (furiner) [inline Kumi.Game/KumiGame.cs]: What

File: Kumi.Game/KumiGame.cs [hunk 2]
```diff
@@ -100,6 +114,12 @@ protected override void LoadComplete()
                 }),
             }, () =>
             {
+                ControlOverlay!.Post(new BasicNotification
+                {
+                    Header = "hi",
+                    Message = "nyan"
+                });
```
Comment (furiner) [inline Kumi.Game/KumiGame.cs]: Should probably remove this.

File: Kumi.Game/KumiGame.cs (hunk 1 above)
Comment (Ryooshuu) [reply Kumi.Game/KumiGame.cs]: don't question it lmfao, i'll re-write the loader code to something better.

File: Kumi.Game/Overlays/Control/Notification.cs [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+                    RowDimensions = new[]
+                    {
+                        new Dimension(GridSizeMode.AutoSize),
+                    },
+                    ColumnDimensions = new[]
+                    {
+                        new Dimension(GridSizeMode.AutoSize),
+                        new Dimension(),
+                        new Dimension(GridSizeMode.AutoSize),
+                    },
```
Comment (furiner) [inline Kumi.Game/Overlays/Control/Notification.cs:67]: What's this about? Why is there only 3?

**Review by furiner — CHANGES_REQUESTED** [review]
nya

File: Kumi.Game/Overlays/Control/Notification.cs (hunk 3 above)
Comment (Ryooshuu) [reply Kumi.Game/Overlays/Control/Notification.cs:67]: Icon, Content, Close button.

**Comment by furiner** [comment]
It would be in your best interest to handle `NotificationDispatchPacket` now that it's a thing in master.

**Review by furiner — APPROVED** [review]

**Pull request merged**